core/indexer/interface.go

```diff
@@ -16,3 +21,19 @@ type Indexer interface {
 	IsInterfaceNil() bool
 	IsNilIndexer() bool
 }
+
+// databaseHandler --
+type databaseHandler interface {
+	saveHeader(header data.HeaderHandler, signersIndexes []uint64)
```

Comment:
These interfaces should have the methods exported (method can remain unexported if they are irrelevant to the other packages). The reason behind this is that the mocks can be placed inside package/mock folder (to comply with the rest of the project) and the unit tests will become more lightweight.

Reply:
done.